src/elements/content-uploader/__tests__/ItemRemove.test.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import noop from 'lodash/noop';
+
+import PlainButton from '../../../components/plain-button';
+import { ItemRemoveBase as ItemRemove } from '../ItemRemove';
+
+import { STATUS_IN_PROGRESS } from '../../../constants';
+
+describe('elements/content-uploader/ItemRemove', () => {
+    test('should have aria-label "Remove" and no aria-describedby', () => {
+        const wrapper = shallow(
+            <ItemRemove
+                intl={{ formatMessage: data => data.defaultMessage }}
+                isResumableUploadsEnabled
```

Comment:
is `isResumableUploadsEnabled` part of this component?

Reply:
oops no it is not, that is leftover from testing. will remove